### ISSUE TITLE
Running into issue getting niceview to work on right half

### ISSUE THREAD

**Issue by yoxalld** [db-ok/SofleChocWireless#7]
**Kind of issue?**
_HELP!_

**What steps have you taken**
* [x] I have searched the Issues page
* [x] I have read the documentation 
* [x] I have swapped displays on each half to make sure they are both functional
* [x] I have reset both halves to ensure display bug from docs isn't an issue

**What is the issue?**
I have built a version of the sofle choc wireless and have been using it successfully for a few months now (Love it so far!). I built a version with no rotary encoders and no rgb. I originally only installed a niceview on the left side. I am now trying to set up the niceview on the right side as well and I am unable to get it to work. I read the build guide and used the example config for the niceviews from your docs. Here is the config I have uploaded [https://github.com/yoxalld/zmk-config-soflechocwireless](url). I'm not sure if I have missed a detail in the config or if there is some other issue.I can't see any issues with connections on board but have added photos for reference just in case. The board still functions normally, just can't get the display on the right half to work. Any insight or guidance you could provide would be greatly appreciated! Thank you for all your work on this project! 

![IMG_3671](https://github.com/db-ok/SofleChocWireless/assets/1072778/c94e265a-b779-4034-a11f-a15c1c6cf945)
![IMG_3672](https://github.com/db-ok/SofleChocWireless/assets/1072778/489c1013-1183-4b1b-acc4-4185085a44a4)
![IMG_3673](https://github.com/db-ok/SofleChocWireless/assets/1072778/4d252656-9a99-4dd1-a0f3-01d67a23c043)
![IMG_3674](https://github.com/db-ok/SofleChocWireless/assets/1072778/54e3e7bf-74b8-4df0-987a-450407b3812d)

**What you expected vs. what you observed:**

* [x] Expected new display to function after flashing new firmware vs. Display on right half stays blank.

**Comment by db-ok** [comment]
I'm glad you tried out this build! I have a version of the PCB that was manufactured from the latest files available in the repo and the nice!view works on the right half, so I'm hopeful there's something that will get this fixed on yours.

Have you tried swapping the display modules between left and right halves to confirm if both display modules are functioning normally?

Another thing you can try is reflowing the header pins for the display. It's a little hard to tell from the photos if there's anything obvious about the solder joints that could be a problem, but reflowing on the PCB and on the display module couldn't hurt. Just be careful with the headers on the display module, as the display itself is apparently somewhat sensitive to prolonged heat. The reason it's hard to tell based on the photos if there's anything wrong with the joints on those headers is that I can't see any of the pin poking through, so it's hard to tell if the joint is good and solid between the thru-hole and the pin. Normally you get a cone shape made by the solder on a through-hole joint. Rounded joints like in your photos are obviously smoother and less likely to catch on something or scratch your desk, but they are not always the best for a reliable connection.

I couldn't see the display jumpers in your photos since they're underneath the microcontroller, but that's another thing to check and make sure they're all bridged on the top side of the PCB. I can tell they're not bridged on the bottom, so that's a good start :) It might be worth just going back over them with a little fresh solder if nothing else works.

Let me know if any of that helps or not!

**Comment by yoxalld** [comment]
Hi, thanks for the response! I have tried swapping the displays and they both work on the left half. For the solder connections, I clipped them with a flush cutter and reflowed them all to be round so they were smooth to the touch. If I get time I can try and reflow them all again.

I did notice that I had forgotten to bridge the display jumpers on the right half at the time I first posted. I must have not bothered when I initially built it because I didn't have a display. So that was obviously my main issue. However, after bridging those connections I was still only getting a few small black dots that would appear and just stay there. I notice a similar effect on the working half right as it turns on, just before the screen refreshes for the initial image. Interestingly though, this morning when I turned it on I got more of an image to appear and you can kind of see the circles from the bluetooth connection widget. I don't know what that means, but it's more progress I guess.

![IMG_3684](https://github.com/db-ok/SofleChocWireless/assets/1072778/1673c449-147a-4dee-853d-f01e76f380f1)

**Comment by yoxalld** [comment]
After re-reading your comment a bunch of times it dawned on me that you mentioned not bridging the jumpers on the bottom. I take it that means what side they are bridged on matters. When I went back and bridged the right side I did it on the bottom because the sockets were in the way on the top. But if they have to be done on the top side to work (which thinking about how the pcb is designed to work flipped would make sense) then maybe that's my issue now. I'll try that and see what I figure out. Thanks!
